pkg/hyper/streaming.go

```diff
@@ -51,17 +72,17 @@ func (h *Runtime) ExecSync(rawContainerID string, cmd []string, timeout int64) (
 	return stdoutBuffer.Bytes(), stderrBuffer.Bytes(), exitCode, nil
 }
 
-// Exec execute a command in the container.
-func (h *Runtime) Exec(rawContainerID string, cmd []string, tty bool, stdin io.Reader, stdout, stderr io.WriteCloser) error {
-	return fmt.Errorf("Not implemented")
+// Exec prepares a streaming endpoint to execute a command in the container.
+func (h *Runtime) Exec(req *kubeapi.ExecRequest) (*kubeapi.ExecResponse, error) {
+	return nil, fmt.Errorf("Not implemented")
```

Comment:
Return the endpoint, refer https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/streaming/server.go#L144